ethers-solc/src/cache.rs

```diff
@@ -130,6 +153,59 @@ impl SolFilesCache {
         self.files.values().all(|entry| entry.all_artifacts_exist())
     }
 
+    /// Returns the path to the artifact of the given `(file, contract)` pair
+    ///
+    /// # Example
+    ///
+    /// ```
+    /// # fn t() {
+    /// use ethers_solc::cache::SolFilesCache;
+    /// use ethers_solc::Project;
+    ///
+    /// let project = Project::builder().build().unwrap();
+    /// let cache = SolFilesCache::read_joined(&project.paths).unwrap();
+    /// cache.find_artifact_path("/.../src/Greeter.sol", "Greeter");
```

Comment:
Are 3 dots valid here!

Reply:
was my poor attempt to illustrate that this is supposed to be an absolute path.
will make the more clear